charts/scalardb/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: scalardb
+description: A Helm chart of Scalar DB
+
+type: application
+version: 2.0.0
```

Comment:
Other charts are using `version: 2.0.0` so I put the same for scalardb? Should I remove it?

Reply:
This is the version of the chart.
So, I think it should be 1.0.0 if it is an initial version.
(other charts use 2.0.0 because they are upgraded. )
https://helm.sh/docs/topics/charts/#charts-and-versioning

Reply:
I got it. Thank you!